### PR TITLE
Add Selenium tests for popup

### PR DESCRIPTION
**Pull request by Hubbacca**

### CODE REVIEW

File: test/selenium-tests/index.po.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+    buttonConfirmSelector() {
+        return By.id('confirm')
+    }
+
+    warrantyTableSelector() {
+        return By.css('.warranty-table');
+    }
+
+
```
Comment (ladral) [inline test/selenium-tests/index.po.js]: Remove one empty line

File: test/selenium-tests/index.test.js [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+            await driver.wait(until.elementLocated(page.machineEntrySelector()));
+            await driver.wait(until.elementLocated(page.employeeEntrySelector()));
+            await driver.wait(until.elementLocated(page.dateEntrySelector()));
+            await driver.wait(until.elementLocated(page.timeEntrySelector()));
+            await driver.wait(until.elementLocated(page.buttonConfirmSelector()));
+
+            await driver.findElement(page.machineEntrySelector()).sendKeys('machine');
+            await driver.findElement(page.employeeEntrySelector()).sendKeys('employee');
+            await driver.findElement(page.dateEntrySelector()).click();
+            await driver.findElement(page.calendarDaySelector(1)).click();
```
Comment (ladral) [inline test/selenium-tests/index.test.js]: Use a day in the middle of the date picker. That way you are sure that the picked day is within the current month.

File: test/selenium-tests/index.po.js [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+const {By} = webdriver
+
+export default class Index{
+    constructor(driver, url){
+        this.driver = driver
+        this.url = url
+    }
+
+    gotoIndex(){
+        return this.driver.navigate().to(this.url)
```
Comment (ladral) [inline test/selenium-tests/index.po.js]: add semicolon at the end of each statement

File: test/selenium-tests/index.test.js [hunk 4]
```diff
@@ -0,0 +1,219 @@
...
+           const timeEntry = await driver.findElement(page.warrantyTableEntriesSelector(7)).getText();
+
+           assert.notEqual(dateEntry, '');
+           assert.equal(customerEntry, 'customer');
+           assert.equal(contactEntry, 'contact');
+           assert.equal(machineEntry, 'machine');
+           assert.equal(issueEntry, 'issue');
+           assert.equal(employeeEntry, 'employee');
+           assert.equal(timeEntry, '01:00:00');
+       })*/
```
Comment (ladral) [inline test/selenium-tests/index.test.js]: copy this test in a separate branch so we can merge this pull request

File: test/selenium-tests/index.po.js (hunk 1 above)
Comment (Hubbacca) [reply test/selenium-tests/index.po.js]: updated

File: test/selenium-tests/index.test.js (hunk 2 above)
Comment (Hubbacca) [reply test/selenium-tests/index.test.js]: updated

File: test/selenium-tests/index.po.js (hunk 3 above)
Comment (Hubbacca) [reply test/selenium-tests/index.po.js]: updated

File: test/selenium-tests/index.test.js (hunk 4 above)
Comment (Hubbacca) [reply test/selenium-tests/index.test.js]: updated

**Comment by Hubbacca** [comment]
This branch is yet ready to merge in develop.

**Pull request merged**